d2target/d2target.go

```diff
@@ -44,10 +44,10 @@ func (diagram Diagram) HashID() (string, error) {
 }
 
 func (diagram Diagram) BoundingBox() (topLeft, bottomRight Point) {
-	x1 := int(math.MaxInt64)
-	y1 := int(math.MaxInt64)
-	x2 := int(-math.MaxInt64)
-	y2 := int(-math.MaxInt64)
+	x1 := int(math.MaxInt32)
+	y1 := int(math.MaxInt32)
+	x2 := int(-math.MaxInt32)
+	y2 := int(-math.MaxInt32)
```

Comment:
looks like `MinInt32` is a thing. can u switch to that?

Reply:
Made the change